src/apprt/gtk/App.zig

```diff
@@ -146,6 +146,9 @@ pub fn init(core_app: *CoreApp, opts: Options) !App {
 
     var gdk_disable: struct {
         @"gles-api": bool = false,
+        /// current gtk implementation for color management is not good enough. 
+        /// see: https://bugs.kde.org/show_bug.cgi?id=495647
+        @"color-mgmt": bool = true,
```

Comment:
I don't think this should be an unconditional workaround. Maybe restrict it to certain GTK versions?

Reply:
Seems like it's GTK's impl and not a bug introduce in a specific version.

Reply:
From the upstream issue it reads that GTK's implementation was broken and is intended to be fixed in future versions. Therefore I would recommend to only disable color management explicitly for those broken GTK versions (IIRC 4.16 was the series that introduced color management, and from there you can track down the patch releases)

Reply:
Rather than tracking down version numbers, I'd just default it to false and add a config option to enable it.

Reply:
I ended up switching to config